packages/atlas/src/hooks/useSegmentAnalytics.ts

```diff
@@ -47,14 +54,57 @@ export const useSegmentAnalytics = () => {
     [analytics]
   )
 
-  const trackVideoView = useCallback(
-    (videoId: string, channelId: string, channelTitle: string, description: string, isNft: boolean) => {
-      analytics.track('video viewed', {
+  const trackVideoPlaybackStarted = useCallback(
+    (videoId: string, channelId: string, title: string, totalLength: number, fullScreen: boolean, quality: string) => {
+      analytics.track('video playback started', {
         videoId,
         channelId,
-        channelTitle,
-        description,
-        isNft,
+        title,
+        totalLength,
+        fullScreen,
+        quality,
+      })
+    },
+    [analytics]
+  )
+
+  const trackVideoPlaybackPaused = useCallback(
+    (videoId: string, channelId: string, title: string, totalLength: number, fullScreen: boolean, quality: string) => {
```

Comment:
I would avoid functions that have more than 3 or 4 parameters. It's bug-prone, you could easily pass the arguments in the wrong order(especially if most of them are in the same type). I would create a function that receives only one object with properties - videoId, channelId, title... etc